react/src/pages/Pogoda_szuk.js

```diff
@@ -1,24 +1,59 @@
 import Navbar from '../components/Navbar';
 import Header from '../components/Header';
 import ButtonsPogoda from '../components/ButtonsPogoda';
+import Search from '../components/Weather/Search';
 import CurrentWeather from '../components/Weather/CurrentWeather';
+import { weatherApi } from '../components/Weather/Api';
+import { useState } from 'react';
+
 function PogodaSzukPage(){
-    return (
-        <div className="App">
-          <div id="header"> 
-            <Header />
-          </div>
-          <div id="menu">
-            <Navbar />
-          </div>
-          <div>
-            <ButtonsPogoda />
-          </div>
-          <div id="weather">
-            <CurrentWeather />
-            </div>
-        </div>
-      );
+
+  const [currentWeather, setCurrentWeather] = useState(null);
+  const [forecast, setForecast] = useState(null);
+
+
+  const handleOnSearchChange = (searchData) => {
```

Comment:
Czy jedynym zadaniem ten funkcji jest szukanie pogody? Jeśli tak, to powinniśmy to nazwać `handleSearch`. `onChange` używamy raczej w przypadku np. zmieniających się wartości w polach tekstowych.